examples/external-data/src/mock-customer-service/service.ts

```diff
@@ -12,29 +12,64 @@ import fetch from "node-fetch";
 import { ClientManager } from "../utilities";
 import { assertValidTaskData, ITaskData } from "../model-interface";
 
+/**
+ * Expected shape of the "broadcast-signal" message that is sent to the /broadcast-signal service endpoint.
+ */
+export interface BroadcastSignalBodyInterface {
+	/**
+	 * Content of signal. Required by server.
+	 */
+	signalContent: {
+		/**
+		 * Required parameter. Required by server. User may add more parameters than the required `type` and `content` below.
+		 */
+		contents: {
+			/**
+			 * Required parameter. User defined content. User defined content that will be pass
+			 * unchanged to client.
+			 */
+			content: JSON | string;
+			/**
+			 * Required parameter. User defined content that will be pass unchanged to client.
+			 * User may consider using this type to differentiate between multiple diffrent signal types,
+			 * and possibly use it for versioning as well.
+			 */
+			type: string;
+		};
```

Comment:
Just a bit of rewording
```suggestion
		/**
		 * Required. User may add more properties besides `type` and `content` below.
		 */
		contents: {
			/**
			 * Required. User defined content that will be passed unchanged to client.
			 */
			content: JSON | string;
			/**
			 * Required. User defined content that will be passed unchanged to client.
			 * User may consider using this type to differentiate between different signal types,
			 * and possibly use it for versioning as well.
			 */
			type: string;
		};
```